src/Aspire.Dashboard/Components/Layout/MainLayout.razor.cs

```diff
@@ -195,8 +195,8 @@ public async Task LaunchSettingsAsync()
         DialogParameters parameters = new()
         {
             Title = Loc[nameof(Resources.Layout.MainLayoutSettingsDialogTitle)],
-            PrimaryAction = Loc[nameof(Resources.Layout.MainLayoutSettingsDialogClose)],
-            PrimaryActionEnabled = true,
+            PrimaryAction = ViewportInformation.IsDesktop ? Loc[nameof(Resources.Layout.MainLayoutSettingsDialogClose)].Value : null,
+            PrimaryActionEnabled = ViewportInformation.IsDesktop,
```

Comment:
Is it consistent with other dialogs not to show a close button in mobile view?

Reply:
The text visualizer also does not have a close button (in either view). The difference here is that on desktop, this is rendered as a popover, instead of a dialog.